pulp_ansible/tests/functional/api/collection/v3/test_namespace.py

```diff
@@ -41,6 +44,336 @@ def _random_image():
     return _random_image
 
 
+def test_invalid_namespace_names(
+    ansible_repo_and_distro_factory,
+    galaxy_v3_plugin_namespaces_api_client,
+    global_ansible_namespaces_api_client,
+):
+    """Test that users can't create namespaces with invalid names."""
+    repo, distro = ansible_repo_and_distro_factory()
+    kwargs = {"path": distro.base_path, "distro_base_path": distro.base_path}
+
+    invalid_names = [
+        "",
+        "_hello_world",
+        "123345",
+        "2cool4school",
+        "x",
+        "xy",
+        "my-namespace-name",
+        "my.namespace.name",
+        "hello!",
+    ]
+
+    for name in invalid_names:
+        try:
```

Comment:
This should use `with pytest.raises`

Reply:
pytest.raises allows only a single exception and this test is checking for ApiException and also ApiValueError.

I am not sure why (I have not written the test) but looks like the API can raise one of those errors, I guess it will depend on the input.

I dont see any advantage of using pytst.raises in this case.

Reply:
Hmm, I'll trust you on that.
But for the record, i believe, pytest.raises is more readable and it marks a test failed, not errored.
Do we really not know which one Exception we are to expect here?